tool/tsh/common/kube_proxy.go

```diff
@@ -647,6 +647,12 @@ kubectl version
 
 // proxyKubeHeadlessTemplate is the message that gets printed to a user when a kube proxy is started with --headless.
 var proxyKubeHeadlessTemplate = template.Must(template.New("").
-	Parse(`Started local proxy for Kubernetes on {{.addr}} in the background
-and kubectl is set up to work with it. Try issuing a command, for example "kubectl version"
-`))
+	Parse(fmt.Sprintf(`Started local proxy for Kubernetes on {{.addr}} in the background.
+
+%v New shell will now be started with kubectl automatically set up to work with the local proxy.
+When you are done, you will need to close this internal shell by issuing "exit" command - that
+will return you to your original shell and all certificates will be automatically cleaned up.
+
+To use different contexts use "kubectl --context='example' ...".
+Try issuing a command, for example "kubectl version".
```

Comment:
Started local proxy for Kubernetes Access in the background.

*Warning*: Teleport will initiate a new shell configured with kubectl for local proxy access. To conclude the session, simply use the **"exit"** command. Upon exiting, your original shell will be restored, the local proxy will be closed, and future access through this headless session won't be possible.

if len(contexts)>0:
   To use different contexts use "kubectl --context='example' ...

Please go ahead and test a command, such as "kubectl version."
 


@greedy52 @AntonAM what do you think?

Reply:
Yep, looks good, changing 👍  (leaving "try issuing..." part as discussed offline)